app/models/manageiq/providers/openstack/network_manager/cloud_network.rb

```diff
@@ -50,7 +50,13 @@ def self.raw_create_cloud_network(ext_management_system, options)
     {:ems_ref => network.id, :name => options[:name]}
   rescue => e
     _log.error "network=[#{options[:name]}], error: #{e}"
-    raise MiqException::MiqNetworkCreateError, parse_error_message_from_neutron_response(e), e.backtrace
+    parsed_error = parse_error_message_from_neutron_response(e)
+    if parsed_error =~ /Invalid input for operation: network_type value '.*' not supported\./
+      error_message = _("Network type '#{options[:provider_network_type]}' is not supported by the Provider")
+      raise MiqException::MiqNetworkCreateError, error_message, e.backtrace
```

Comment:
Looks good! But you could do this without two "raises" by assigning back to `parsed_error` :)

Reply:
Thanks for review, does it look better now?